### PR TITLE
completed sql-select exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: sql-select/actor-forty.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+select *
+  from "actors"
+  where "actorId"= '40';
```
Comment (scbowler) [inline sql-select/actor-forty.sql:3]: Numbers shouldn't be in quotes

File: sql-select/expensive-films.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+select *
+  from "films",
+  order by "replacementCost" asc,
```
Comment (scbowler) [inline sql-select/expensive-films.sql:3]: Here you want the sort order to be `desc`

File: sql-select/inactive-customers.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+select "isActive"
+  from "customers"
+  order by "false" desc;
```
Comment (scbowler) [inline sql-select/inactive-customers.sql:3]: This should be `where "isActive" = false;` No `order by`  needed for this one.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just a couple updates needed. Let me know if you have any questions.

File: sql-select/inactive-customers.sql (hunk 3 above)
Comment (avallonsmith) [reply sql-select/inactive-customers.sql]: image
corrected!

File: sql-select/actor-forty.sql (hunk 1 above)
Comment (avallonsmith) [reply sql-select/actor-forty.sql]: image
corrected!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**